inst/testhubs/v5/target_dir/README.Rmd

```diff
@@ -23,56 +23,63 @@ The example model outputs that are provided here are adapted from forecasts subm
 
 ## Working with the data in R
```

Comment:
I wonder if it would make sense to add a bit of information about `hubUtils` here before the "working with the data" section. This next session is mostly based on `hubData` so it might be good to present the link with hubUtils before, especially as it's the README for hubUtils, if that make sense?

Reply:
As it's for the complex hub example, please ignore